pkg/cmd/opts/install.go

```diff
@@ -170,7 +170,7 @@ func (o *CommonOptions) InstallBrewIfRequired() error {
 		return nil
 	}
 
-	_, flag, err := packages.ShouldInstallBinary("brew")
+	flag, err := packages.ShouldInstallBinary("brew")
```

Comment:
We cannot do that anymore. Now that we removed the lookup in the global PATH in ShouldInstallBinary, we would effectively check whether brew is installed in JX_HOME and if not (and it will never be) we would start the brew install which is going to install brew globally.

Really, we should not install Homebrew to start with, but if we want to keep the existing functionality and going in increments, we would need a new helper function IsInPath which checks whether a given binary is on the global PATH. For Homebrew we would have to call this function then opposed to ShouldInstallBinary. We need to differentiate between the usecase of checking whether we installed something into JX_HOME or globally